src/test/java/com/puppycrawl/tools/checkstyle/AbstractPathTestSupport.java

```diff
@@ -24,11 +24,834 @@
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import com.puppycrawl.tools.checkstyle.api.CheckstyleException;
+import com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheckTest;
+
 public abstract class AbstractPathTestSupport {
 
+    /**
+     * Input files which have more than 120 lines.
+     */
+    public static final Set<String> FILE_SIZE_SUPPRESSIONS = new HashSet<>(Arrays.asList(
+            "src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc"
+                    + "/nonemptyatclausedescription/InputNonEmptyAtclauseDescription.java",
+            "src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocstyle"
+                    + "/InputJavadocStyle2.java",
```

Comment:
The complete path was used as we have files with the same name.
Example - 
https://github.com/checkstyle/checkstyle/blob/master/src/test/resources/com/puppycrawl/tools/checkstyle/grammar/InputRegressionJavaClass1.java
https://github.com/checkstyle/checkstyle/blob/master/src/test/resources/com/puppycrawl/tools/checkstyle/grammar/antlr4/InputRegressionJavaClass1.java